js/script.js

```diff
@@ -0,0 +1,36 @@
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+function getQuote() {
+     $.getJSON(prefix + quoteUrl, createTweet);
+     $.ajaxSetup({ cache: false });
+}
+
+function createTweet(input) {
+    var data = input[0];
```

Comment:
at first check that in response you get at least one element in array :) when not, just stop your function :)